custom_components/miele/translations/de.json

```diff
@@ -417,10 +416,10 @@
                             "dust_box_missing": "Fehlender Staubbehälter",
```

Comment:
<strike>Miele calls it the `Staubbox`, so it would be `Fehlende Staubbox` or `Staubbox fehlt`.</strike>

According to the error message description in the Scout RX3 Home Vision manual: `Filter nicht erkannt`.